apps/desktop-wallet/src/components/TransactionalInfo.tsx

```diff
@@ -84,6 +87,9 @@ const TransactionalInfo = ({
         <CellArrow>
           <TransactionIcon color={iconBgColor}>
             <Icon size={13} strokeWidth={3} color={iconColor} />
+            <FailedTXBubble data-tooltip-id="default" data-tooltip-content={t('Script execution failed')}>
```

Comment:
I think we need to add a condition here :)

Reply:
Ah right, thanks for catching it, I was testing how it looks :D

Reply:
Done!